homeassistant/components/webostv/__init__.py

```diff
@@ -1 +1,213 @@
 """Support for WebOS TV."""
+import asyncio
+import logging
+from websockets.exceptions import ConnectionClosed
```

Comment:
Please add a blank line between standard library and 3rd party imports.

I recommend the isort tool to automatically sort imports.